uicomponent-compose/core/src/main/java/io/github/droidkaigi/feeder/core/theme/Shape.kt

```diff
@@ -1,11 +1,19 @@
 package io.github.droidkaigi.feeder.core.theme
 
-import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.shape.CutCornerShape
 import androidx.compose.material.Shapes
 import androidx.compose.ui.unit.dp
 
 val shapes = Shapes(
-    small = RoundedCornerShape(4.dp),
-    medium = RoundedCornerShape(4.dp),
-    large = RoundedCornerShape(0.dp)
+    small = CutCornerShape(
+        topStart = 8.dp,
+        bottomEnd = 8.dp
+    ),
+    medium = CutCornerShape(
```

Comment:
There was no medium components(ex. Card, Dialog) in Figma, so temporarily made the same as small shape.